AmplifyPlugins/Auth/AWSAuthPlugin/Dependency/AuthenticationProviderAdapter+SignIn.swift

```diff
@@ -74,6 +68,45 @@ extension AuthenticationProviderAdapter {
         }
     }
 
+    func confirmSignIn(request: AuthConfirmSignInRequest,
+                       completionHandler: @escaping (Result<AuthSignInResult, AmplifyAuthError>) -> Void) {
+
+        let userAttributes = (request.options.pluginOptions as? AWSAuthConfirmSignInOptions)?.userAttributes
+        let clientMetaData = (request.options.pluginOptions as? AWSAuthConfirmSignInOptions)?.metadata
+        awsMobileClient.confirmSignIn(challengeResponse: request.challengeResponse,
+                                      userAttributes: [:],
```

Comment:
looks like `userAttributes` at line 74 should be passed in at line 77

Reply:
Added